less/index.less

```diff
@@ -54,6 +54,155 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	min-height: 100%;
 }
 
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+}
+
+// variables
+
+// mix-ins
+
+.section-heading-content() {
+	h1 {
+		font-size: 4.2rem;
+		color: firebrick;
+		font-weight: bold;
+	}
+
+	h2 {
+		font-size: 2.0rem;
+		font-weight: bold;
+		margin: 25px 0 8px 0;
+	}
+
+	h3 {
+		font-size: 1.4rem;
+		margin: 8px 0;
+	}
+
+	p {
+		font-size: 1.4rem;
+		margin: 20px 0;
+	}
+}
+
+// fixed header
+
+header {
+	display: flex;
+	align-items: center;
+	justify-content: flex-end;
+	position: fixed;
+	background-color: black;
+	width: 100%;
+	height: 70px;
+	top: 0;
+	left: 0;
+	z-index: 1;
+	box-shadow: 0 0 30px 0;
+
+	nav {
+		display: flex;
+		justify-content: space-around;
+		width: 30%;
+
+		a {
+			color: white;
+			font-size: 1.8rem;
+			text-decoration: none;
+		}
+
+		a:hover {
+			text-decoration: underline;
+		}
+	}
+}
+
+main {
+	display: flex;
+	height: 100%;
+
+	.side-container {
+		width: 100%;
+		height: 100%;
+		max-width: 350px;
+		background-color: firebrick;
+
+		.avatar {
+			display: flex;
+			justify-content: center;
+			margin: 140px 0 40px 0;
+
+			img {
+				height: 250px;
+				width: 250px;
+			}
+		}
+
+		.name {
+			display: flex;
+			flex-direction: column;
+			align-items: center;
+
+			h1 {
+				font-size: 3.8rem;
+				color: white;
+				margin-top: 15px;
+			}
+
+			h2 {
+				font-size: 1.8rem;
+				color: white;
+				margin-top: 15px;
+			}
+		}
+
+		.get-in-touch {
+			display: flex;
+			flex-direction: column;
+			align-items: center;
+			font-size: 1.4rem;
+			color: white;
+			margin: 100px 0 10px 0;
+
+			.git-icons {
+				display: flex;
+				margin: 20px 0;
+
+				img {
+					height: 50px;
+					width: 50px;
+					padding: 5px;		
+				}
+			}
+		}
+	}
+
+	.main-container {
+		width: 100%;
+		max-width: 600px;
+		margin: 140px 70px;
+
+		.section-content {
+			.section-heading-content();
+
+			.section-h1 {
+					margin-top: 70px;
+			}
+
+			ul {
+
+				li {
+					list-style-type: square;
+					list-style-position: inside;
+					font-size: 1.4rem;
+				}
+			}
+		}
+	}
+}
```

Comment:
Rule of thumb: Never nest more than 4 levels deep. This is because the deeper you go, the more memory it takes to load. Fewer selectors are more performant. 4 Levels Deep is what I was taught while going through the program and it has served me well. Let me know if you have any questions about organizing your stylings, with those limitations in place.